tests/test_helper/bats/bgp_control_plane.bats

```diff
@@ -282,4 +287,23 @@ function bgp_bad_netplan(){
     lxc_exec "$MICROOVN_BGP_CONTAINER" "snap alias microovn.ovs-vsctl ovs-vsctl"
 }
 
+# bgp_no_vrf
+#
+# Simple test that checks that the bgp enable command fails with a relevant
+# error message when the VRF kernel module is not loaded
+function bgp_no_vrf(){
+    lxc_exec "$TEST_VM_NO_VRF" "snap unalias ovs-vsctl"
```

Comment:
If I understand your test case correctly, you are starting a VM for it to ensure you can control presence of kernel vrf support.

Since you already control that, why would you need to do the unalias step? Seems to me we would expect this to fail on the account of missing vrf functionality, and not Netplan's inability of calling `ovs-vsctl`?